src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import 'partials/reset';
```

Comment:
no they work that way http://thesassway.com/beginner/how-to-structure-a-sass-project